src/ixdat/measurements.py

```diff
@@ -76,8 +115,8 @@ def __init__(
         self.technique = technique
         self.metadata = metadata or {}
         self.reader = reader
-        self.plotter = plotter
-        self.exporter = exporter or CSVExporter(measurement=self)
+        self._plotter = plotter
```

Comment:
I think it was to avoid a circular import (but the plotter modules shouldn't have to import `Measurement`...) or to avoid importing matplotlib before it was necessary (a nasty manipulation during Issue #10 frustration). I can't think of any reason now that your suggestion here isn't better. I will try it out.

Reply:
Done. It works fine.